config/schemas/ccr_claim_schema.json

```diff
@@ -110,6 +110,14 @@
             "id": "/properties/bills/items/properties/amount",
             "type": "string"
           },
+          "warrant_issued_date": {
```

Comment:
Having this information flat out on the claim schema, won't it have the same problem we were discussing offline: you can technically claim a warrant fee per defendant (not currently officially supported) ?